app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
+	margin-top: 0
 }
 
-.section_headings:before, .section_headings:after {
```

Comment:
That explanation would be the following from the commit message:
```
 • Remove the mid-line from larger headings and left-align them, as
   centred and left-aligned text should not be mixed (since they don’t
   line up, your eyes have to jump around on the page to read them
   smoothly).
```